.gitignore

```diff
@@ -16,3 +16,4 @@ logs
 
 # swagger-gen
 /tmp-swagger-gen
+mocks
```

Comment:
we are using the files in helper/mocks/ in our test cases right? And that needs to be tracked under code revision.